cg/meta/demultiplex/status_db_storage_functions.py

```diff
@@ -25,28 +25,33 @@
 def store_flow_cell_data_in_status_db(
     parsed_flow_cell: FlowCellDirectoryData, store: Store
 ) -> None:
-    """Create flow cell from the parsed and validated flow cell data."""
-    if not store.get_flow_cell_by_name(flow_cell_name=parsed_flow_cell.id):
+    """
+    Create flow cell from the parsed and validated flow cell data.
+    And add the samples on the flow cell to the model.
+    """
+    flow_cell: Flowcell = store.get_flow_cell_by_name(flow_cell_name=parsed_flow_cell.id)
```

Comment:
type hint should be Optional[Flowcell] as to not lie

Reply:
Also consider skipping the named parameter if black introduces line breaks, since it is only a single parameter it does not really add any value. Alternatively one could also create a variable above `flow_cell_name: str = parsed_flow_cell.id` and use it in the call